ext/opcache/Optimizer/zend_func_info.c

```diff
@@ -732,7 +732,7 @@ static const func_info_t func_infos[] = {
 	F1("str_rot13",                    MAY_BE_STRING),
 	F1("stream_get_filters",           MAY_BE_ARRAY | MAY_BE_ARRAY_KEY_LONG | MAY_BE_ARRAY_OF_STRING),
 	F0("stream_filter_register",       MAY_BE_FALSE | MAY_BE_TRUE),
-	F1("stream_bucket_make_writeable", MAY_BE_FALSE | MAY_BE_OBJECT),
+	F1("stream_bucket_make_writeable", MAY_BE_NULL | MAY_BE_FALSE | MAY_BE_OBJECT),
```

Comment:
While at it, the `false` type can be removed instead.

Reply:
I assume that the return value is unused because zend_fetch_resource throws an exception by calling zend_type_error.

- Are there any circumstances in the interpreter where calling RETURN_FALSE has a side effect even if an Error or Exception is thrown?

```
if ((brigade = (php_stream_bucket_brigade*)zend_fetch_resource(
				Z_RES_P(zbrigade), PHP_STREAM_BRIGADE_RES_NAME, le_bucket_brigade)) == NULL) {
	RETURN_FALSE;
}
```